### PR TITLE
amazonGlacierCmdInterface: add expression

### PR DESCRIPTION
**Pull request by lovek323**
Not sure whether to leave it in top-level `all-packages.nix`, so I'm happy to remove from there if it's not desirable.

### CODE REVIEW

**Comment by pSub** [comment]
`error: Could not find suitable distribution for Requirement.parse('prettytable')`

**Comment by lovek323** [comment]
I am kind of failing here. Anybody help appreciated.

**Comment by offlinehacker** [comment]
Hm, i am eventually co-author of this package, but at some point didn't had time to work on it. Does it still work?

**Comment by lovek323** [comment]
Ha yep. I'm using it. :smile:

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -62,6 +62,8 @@ let
 
   # packages defined elsewhere
 
+  amazonGlacierCmdInterface = callPackage ../tools/backup/amazon-glacier-cmd-interface { };
```
Comment (lethalman) [inline pkgs/top-level/python-packages.nix]: Given it's not a library, I think you can remove it from pythonPackages.

**Comment by domenkozar** [comment]
@lovek323 ping, this is almost ready :-)

**Comment by domenkozar** [comment]
Pushed in 5e28dcc502f348e58a4009b604efe4bbdc74eb8d

**Pull request closed without merging**